deploy-agent/deployd/common/config.py

```diff
@@ -252,7 +252,7 @@ def get_verify_https_certificate(self):
         return self.get_var('verify_https_certificate', 'False')
 
     def get_deploy_agent_version(self):
-        return self.get_var('deploy_agent_version', None)
+        return self.get_var('deploy_agent_version', '1.2.24')
```

Comment:
if we hardcode the version here, do we need to update this everytime we upgrade deploy_agent? 
maybe there is a better way not to hardcode the version here.

Reply:
what's the reason for change in this file in particular? 
Note, this affects Prod not just macos and could give wrong version if not populated.
I think better to default to None/Unknown if not populated instead of populating version which might be wrong.